### PR TITLE
wip: Add feature_last_updated_at to featurestate

### LINKED ISSUE #184: Show "stale" flags
This will help people identity flags that have not changed for a while and are good candidates for removal.

Add a button (somewhere! Not sure where - maybe by the current tags or by the search filter top right) to filter flags that haven't had a state change in N days (suggest N=30 initially but can be changed by the user).

The following events should be considered a state change:

- Creation of flag
- Modifying core flag state (boolean, config, MV)
- Adding/removing/changing segment override
- Adding/removing Identity override

## UI

Add a "Last Modified" option here: 

image

### PR DESCRIPTION
**Pull request by gagantrivedi**

### CODE REVIEW

File: api/features/serializers.py [hunk 1]
```diff
@@ -141,11 +142,20 @@ class FeatureStateSerializerBasic(WritableNestedModelSerializer):
     multivariate_feature_state_values = MultivariateFeatureStateValueSerializer(
         many=True, required=False
     )
+    # TODO: better variable name
+    feature_last_updated_since = serializers.SerializerMethodField()
```
Comment (gagantrivedi) [inline api/features/serializers.py]: @matthewelwell  can you help with the name?

Comment (matthewelwell) [reply api/features/serializers.py]: Is this a datetime or an integer?

If datetime: `feature_last_updated_at`

If integer: `days_since_feature_last_updated`

Comment (gagantrivedi) [reply api/features/serializers.py]: yeah, it's an integer.

Comment (matthewelwell) [reply api/features/serializers.py]: Let's go with `days_since_feature_last_updated` then.

**Pull request closed without merging**